test/functionalTest/cases/1705_csub_cache_objects/csub_cache_objects.test

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
As "remark note": a .test is missing in this PR (we have been discussing about it at skype today but I'm unable right now of remembering the topic of the .test :)